sys/net/gnrc/netif/gnrc_netif.c

```diff
@@ -892,7 +892,12 @@ static int _match_to_idx(const gnrc_netif_t *netif,
 
 static uint8_t _get_scope(const ipv6_addr_t *addr)
 {
-    if (ipv6_addr_is_link_local(addr)) {
+    if (ipv6_addr_is_multicast(addr)) {
```

Comment:
I wanted a fixup. Mainly, because it is a bit beyond the original intention of this PR and I wanted to here your opinion first. I can make it its own commit later-on still.

Reply:
I think it should be a separate commit. I don't mind including it, its needed for multicast but in a different commit.